### PR TITLE
Fix readme.md and index.html for deferred-banner sample.

### PR DESCRIPTION
**Pull request by markafoltz**
- Fix link to chromestatus
- Correct text about what happens with the flag
- Correct text about what happens with the button
- Fix URL to demo

### CODE REVIEW

File: app-install-banner/deferred-banner/readme.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+See https://googlechrome.github.io/samples/app-install-banner/deferred-banner/index.html for a live demo.
 
 
 #### Directions:
 
-* Install Chrome, or another browser that supports Service Workers and manifest
-* Visit the site over two separate days.
-* Or, more usefully, enable chrome://flags/#bypass-app-banner-engagement-checks
+* Install Chrome, or another browser that supports Service Workers and manifest.
+* Wait a few minutes.
```
Comment (markafoltz) [inline app-install-banner/deferred-banner/readme.md]: @beaufortfrancois I did not have to wait for the button to be enabled on my local server (ChromeOS)

File: app-install-banner/deferred-banner/index.html [hunk 2]
```diff
@@ -42,16 +42,18 @@ <h1>Web App Install Banner: Defer prompt of banner sample</h1>
        This API and example shows you how you can defer the prompt until a later time, in this case when the 
        user clicks the button below.  If the button below is disabled, Chrome has chosen now is not a good time to 
        show the prompt.</p>
+
+    <p>If the button is disabled, come back in a few minutes and the button will be enabled.</p>
+
+    <p>For testing, we encourage you to force the button to be enabled by setting the
```
Comment (beaufortfrancois) [inline app-install-banner/deferred-banner/index.html]: We may want to wait a bit before updating this sample as @dmurph is removing the engagement requirement for the `beforeinstallprompt` event in Chrome 131

File: app-install-banner/deferred-banner/readme.md (hunk 1 above)
Comment (beaufortfrancois) [inline app-install-banner/deferred-banner/readme.md]: I wonder if it's still true... @dmurph may know

File: app-install-banner/deferred-banner/readme.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+* Or, enable chrome://flags/#bypass-app-banner-engagement-checks to enable the button right away.
 
-On the second vist, you will see a message on the page that says the user either installed the app or dismissed the banner.
\ No newline at end of file
+Once the button is enabled, Chrome will display the Add to Home Screen prompt when you click it.
```
Comment (beaufortfrancois) [inline app-install-banner/deferred-banner/readme.md]: "Add to Home Screen" is for mobile. On Desktop, it's called "Install App". Maybe use "Chrome will trigger the install flow when you click it"?

File: app-install-banner/deferred-banner/readme.md (hunk 1 above)
Comment (dmurph) [reply app-install-banner/deferred-banner/readme.md]: You shouldn't need to wait, it should just promote right away if you have the other requirements (display set, start_url set, icon that satisfies the requirement, etc)

Comment (dmurph) [reply app-install-banner/deferred-banner/readme.md]: Also - ServiceWorkers are no longer required in Chrome (maybe also Chromium in general, IDK about Edge) for promotion.

File: app-install-banner/deferred-banner/readme.md (hunk 3 above)
Comment (markafoltz) [reply app-install-banner/deferred-banner/readme.md]: Updated these files to refer to "install flow"

File: app-install-banner/deferred-banner/readme.md (hunk 1 above)
Comment (markafoltz) [reply app-install-banner/deferred-banner/readme.md]: Removed references to service workers, the flag, and waiting for the button to be activated

**Review by beaufortfrancois — APPROVED** [review]

**Pull request merged**